src/object-identifier.scss

```diff
@@ -0,0 +1,61 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-object-identifier;
+
+.#{$block} {
+  @mixin fd-object-identifier-title-base {
+    @include fd-reset();
+
+    color: var(--sapList_TextColor);
+  }
+
+  $fd-fd-object-identifier-title-font-size: var(--sapFontLargeSize);
+  $fd-fd-object-identifier-title-font-weight: bold;
+
+  &__title,
+  &__title-bold {
+    @include fd-object-identifier-title-base();
+  }
+
+  &__link {
+    @include fd-reset();
+    @include fd-link();
+    @include action-cursor();
+  }
+
+  /* stylelint-disable no-duplicate-selectors */
+  &__title,
+  &__title-bold {
+    font-size: $fd-fd-object-identifier-title-font-size;
+
+    a {
```

Comment:
Is it needed to style by `a` tag?
It looks like you expect users to compose this component like:

```
<h4 class="fd-object-identifier__title">
    <a>Title</a>
</h4>
```

But in examples you use it like:
```
<p class="fd-object-identifier__title fd-object-identifier__link">Title</p>
```

Reply:
Yes, I expect that users can compose as you brought the example
and I have the example for it
```html
<div class="fd-object-identifier">
    <p class="fd-object-identifier__title"><a href="#" class="fd-link">NoteBook Basic 15</a></p>
</div>
```